wikitemplate-tor-macOS(Intel).md

```diff
@@ -0,0 +1,79 @@
+### **`macOS 11.0 Big Sur`**
+
+### Tor Client Updater 
+
+- [ ] For development go-update-server. Run brave-browser with `--user-data-dir=component-dev --use-dev-goupdater-url` (These flags are only available in v1.7.x). Once the crx is pushed to production run without these flags.
+- [ ] Navigate to `brave://components` and verify `Tor Client Updater (OS)` is updated successfully.
+- [ ] Open `New Private Window with Tor` and confirm that it starts without any errors.
+	- [ ] Navigate to `check.torproject.org` and verify that tor is working successfully.
+	- [ ] Navigate to `brave.com` and `http://3g2upl4pq6kufc4m.onion/` to check if the sites work correctly.
+- [ ] Load `brave.com` and `mail.protonmail.com` in a regular Window/Tab and ensure that clicking on `Open in Tor` correctly launches a Tor window and opens the appropriate `.onion` website.
+
+### MacOS
+- [ ] Navigate to `/Users/<user>/Library/Application Support/BraveSoftware/Brave-Browser-<channel>/cldoidikboihgcjfkhdeidbpclkineef/<version>`
+- [ ] Run `codesign -vvvv tor-<version-tor>-darwin-brave-<version-brave>` to confirm codesign is valid
+- [ ] For MacOS Catalina (10.15+) - Run `spctl -a -vv -t install tor-<version-tor>-darwin-brave-<version-brave>` to verify that the binary is notarized.
+
```

Comment:
Not sure if copy/paste error, but `Catalina` isn't listed, should be between `Big Sur` and `Mojave`

Reply:
That was copy paste from one of the test runs.